quick-search-engine/README.md

```diff
@@ -0,0 +1,116 @@
+# Qdrant Search Engine
+
+This code contains code and instructions for building a search engine using Qdrant. Qdrant is particularly well-suited for tasks involving similarity search, such as recommendation systems, image or text retrieval, and clustering.
+
+## Article Overview
+
+The associated Medium article titled ["Build a search engine in 5 minutes using Qdrant"](https://medium.com/@armaghanshakir/build-a-search-engine-in-5-minutes-using-qdrant-5c7a3ef6ac5) provides a step-by-step guide on building a search engine using Qdrant.
+
+## Instructions
+
+### Setting Up
+
+1. Join the [Quora Question Pairs Competition on Kaggle](https://www.kaggle.com/c/quora-question-pairs).
+2. Download the file `train.csv.zip`.
+3. Unzip the downloaded file.
+4. Save the path to the dataset in the `DATA_PATH` variable.
```

Comment:
I've a minor preference for using the Kaggle API or the same dataset from Huggingface dataset — that way, everything can be done via code and we don't need to manually set paths or other things outside of our code. What do you think?

Reply:
The problem with using kaggle API here is that it would add many steps and some confusion, for the sake of simplicity I have kept it like this

Reply:
Please use this instead? https://huggingface.co/datasets/quora

Assuming they're identical